code/go/0chain.net/smartcontract/storagesc/allocation_test.go

```diff
@@ -1390,8 +1390,7 @@ func Test_finalize_allocation(t *testing.T) {
 	var tx = newTransaction(b1.id, ssc.ID, 0, tp)
 	balances.setTransaction(t, tx)
 	var resp string
-	resp, err = ssc.commitBlobberConnection(tx, mustEncode(t, &cc),
-		balances)
```

Comment:
As there is no meaninful change and its nothing to do with zcnsc, its probably  best to remove allocation_test.go from the PR.

Reply:
Yes, just strange formatting, I changed it accidentally